package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mount-react-components",
-  "version": "1.1.3",
+  "version": "1.2.0",
```

Comment:
Regarding how do we handle one version of the package to be compatible with one version (or a newer version of react) as discussed today in our slack, I think we should at least match the major number with react's major number (see react and react-dom matching their versions/releases). This will make things simple for us. Based on the react version we are running, that's the mount-react-components version we should run.

Thoughts?

Reply:
Hi @patriciomacadden ! Sorry for the late response (1 year lol).
Totally agree, I already update the version number in this PR.